test/article.js

```diff
@@ -17,6 +17,19 @@ chai.use(chaiHttp);
 
 let validToken, slug, unverifiedToken;
 let updatedSlug, invalidToken;
+
+describe('Unavailable URL', () => {
+  it('Should return an error message, Once you tried to access the unavaliable URL', () => {
```

Comment:
this test does not belong in this file. Instead you should create a `server.test.js` file for it.

Reply:
that is good idea @deschantkn